pkg/installmanager/installmanager.go

```diff
@@ -1133,6 +1147,19 @@ func waitForProvisioningStage(provision *hivev1.ClusterProvision, m *InstallMana
 	return errors.Wrap(err, "ClusterProvision did not transition to provisioning stage")
 }
 
+func (m *InstallManager) setupSSHKnownHost(knownHost string) error {
```

Comment:
Long term, this function should be resilient to the directory and the file already existing.

Reply:
The file write should overwrite as it was, and I switched to os.MkdirAll which should return nothing in the event of the dir(s) already existing.

Reply:
By resilient to an already existing file, I meant that it should *not* overwrite the file. It should only add the additional known host to the file. But that is a problem that we can address another day.